src/Orleans/Runtime/RequestContext.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
+using System.Threading;
 #if !NETSTANDARD
 using System.Runtime.Remoting.Messaging;
 #else
```

Comment:
check this ifdefing (you are adding System.Threading two times, one outside the `if`, and one inside the `else`)